lib/db/feature-tag-store.js

```diff
@@ -0,0 +1,245 @@
+'use strict';
+
+const metricsHelper = require('../metrics-helper');
+const { DB_TIME } = require('../events');
+const {
+    TAG_CREATED,
+    TAG_DELETED,
+    FEATURE_TAGGED,
+    FEATURE_UNTAGGED,
+} = require('../event-type');
+const { CREATE_TAG, DELETE_TAG } = require('../command-type');
+const NotFoundError = require('../error/notfound-error');
+
+const COLUMNS = ['id', 'type', 'value'];
+const TABLE = 'tags';
+const FEATURE_TAG_TABLE = 'feature_tag';
+
+class FeatureTagStore {
+    constructor(db, eventStore, eventBus, getLogger) {
+        this.db = db;
+        this.eventStore = eventStore;
```

Comment:
it's fine for now, but in the upcoming cleanup where we introduce a "service layer" encapsulating more of the logic, i think this store should not need to know about the eventStore at all.

Reply:
That would be my hope as well, actually goes for all our stores.